certified-connectors/SMSLink/apiDefinition.swagger.json

```diff
@@ -0,0 +1,89 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SMSLink",
+    "description": "SMSLink is a SMS provider that provides integrated services such as SMS Marketing, SMS Gateway and Mail to SMS.",
+    "version": "1.0"
+  },

```

Comment:
@smslink Please add "contact" section to info

Reply:
Done. Please note that contact section is not well documented, most connectors don't have this section, so we had to search one connector and use the syntax from there.